### PR TITLE
Fix amp-pan-zoom zoom target bug and relayout bug

### PR DESCRIPTION
**Pull request by cathyxz**
Bug 1: The tap target x,y coordinates for zoom are taken relative to viewport instead of relative to element. Fixed by subtracting the viewport x,y coordinates. 
Bug 2: Relayout shouldn't be happening on resize.

### CODE REVIEW

File: extensions/amp-pan-zoom/0.1/amp-pan-zoom.js [hunk 1]
```diff
@@ -324,6 +321,26 @@ export class AmpPanZoom extends AMP.BaseElement {
...
   }
 
+  /**
+   * Given a x offset relative to the viewport, return the x offset
+   * relative to the amp-pan-zoom component.
+   * @param {number} clientX
+   */
+  getOffsetX_(clientX) {
+    return (this.elementBox_.left - this.getViewport().getScrollLeft())
+      - clientX;
```
Comment (cvializ) [inline extensions/amp-pan-zoom/0.1/amp-pan-zoom.js]: mini-nit: inconsistent indentation with the function below

Comment (cvializ) [reply extensions/amp-pan-zoom/0.1/amp-pan-zoom.js]: The dangling `- variable` slows down the read through a little. Consider using an intermediate variable to store and name the `this.elementBox_.left - this.getViewport().getScrollLeft()` computation for clarity. A long line can be a symptom indicating that too much is happening in one line.

File: extensions/amp-pan-zoom/0.1/amp-pan-zoom.js [hunk 2]
```diff
@@ -181,9 +181,11 @@ export class AmpPanZoom extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
-    this.content_.classList.add('i-amphtml-pan-zoom-child');
-    return this.resetContentDimensions_()
-        .then(() => this.setupGestures_());
+    return this.resetContentDimensions_().then(() => {
+      if (!this.gestures_) {
```
Comment (cvializ) [inline extensions/amp-pan-zoom/0.1/amp-pan-zoom.js]: nit: [prefer positive if conditions ](http://go/tott-530)(internal link) and shortcut `return`s

**Comment by cathyxz** [comment]
Sorry there was a bit of a race condition here on github between CI passing and your comments. I'll address these in a follow up PR.

**Pull request merged**